runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -0,0 +1,244 @@
+
+	// Infinity colour kits
+	ITEM_INFINITY_TOP(INFINITY_TOP, INFINITY_TOP_10605, INFINITY_TOP_20574, DARK_INFINITY_TOP, LIGHT_INFINITY_TOP),
+	ITEM_INFINITY_TOP_LIGHT_COLOUR_KIT(LIGHT_INFINITY_COLOUR_KIT, LIGHT_INFINITY_TOP),
+	ITEM_INFINITY_TOP_DARK_COLOUR_KIT(DARK_INFINITY_COLOUR_KIT, DARK_INFINITY_TOP),
+	ITEM_INFINITY_BOTTOMS(INFINITY_BOTTOMS, INFINITY_BOTTOMS_20575, DARK_INFINITY_BOTTOMS, LIGHT_INFINITY_BOTTOMS),
+	ITEM_INFINITY_BOTTOMS_LIGHT_COLOUR_KIT(LIGHT_INFINITY_COLOUR_KIT, LIGHT_INFINITY_BOTTOMS),
+	ITEM_INFINITY_BOTTOMS_DARK_COLOUR_KIT(DARK_INFINITY_COLOUR_KIT, DARK_INFINITY_BOTTOMS),
+	ITEM_INFINITY_HAT(INFINITY_HAT, DARK_INFINITY_HAT, LIGHT_INFINITY_HAT),
+	ITEM_INFINITY_HAT_LIGHT_COLOUR_KIT(LIGHT_INFINITY_COLOUR_KIT, LIGHT_INFINITY_HAT),
+	ITEM_INFINITY_HAT_DARK_COLOUR_KIT(DARK_INFINITY_COLOUR_KIT, DARK_INFINITY_HAT),
+
+	// Miscellaneous ornament kits
+	ITEM_DARK_BOW(DARK_BOW, DARK_BOW_12765, DARK_BOW_12766, DARK_BOW_12767, DARK_BOW_12768, DARK_BOW_20408),
+	ITEM_ODIUM_WARD(ODIUM_WARD, ODIUM_WARD_12807),
+	ITEM_MALEDICTION_WARD(MALEDICTION_WARD, MALEDICTION_WARD_12806),
+	ITEM_STEAM_BATTLESTAFF(STEAM_BATTLESTAFF, STEAM_BATTLESTAFF_12795),
+	ITEM_LAVA_BATTLESTAFF(LAVA_BATTLESTAFF, LAVA_BATTLESTAFF_21198),
+
+	// Slayer helm/black mask
+	ITEM_BLACK_MASK(
+		BLACK_MASK, BLACK_MASK_I, BLACK_MASK_1, BLACK_MASK_1_I, BLACK_MASK_2, BLACK_MASK_2_I, BLACK_MASK_3, BLACK_MASK_3_I, BLACK_MASK_4, BLACK_MASK_4_I, BLACK_MASK_5,
+		BLACK_MASK_5_I, BLACK_MASK_6, BLACK_MASK_6_I, BLACK_MASK_7, BLACK_MASK_7_I, BLACK_MASK_8, BLACK_MASK_8_I, BLACK_MASK_9, BLACK_MASK_9_I, BLACK_MASK_10_I,
+		SLAYER_HELMET, SLAYER_HELMET_I, BLACK_SLAYER_HELMET, BLACK_SLAYER_HELMET_I, PURPLE_SLAYER_HELMET, PURPLE_SLAYER_HELMET_I, RED_SLAYER_HELMET, RED_SLAYER_HELMET_I,
+		GREEN_SLAYER_HELMET, GREEN_SLAYER_HELMET_I, TURQUOISE_SLAYER_HELMET, TURQUOISE_SLAYER_HELMET_I);
+
+	private static final Multimap<Integer, Integer> MAPPINGS = HashMultimap.create();
+	private final int tradeableItem;
+	private final int[] untradableItems;
+
+	static
+	{
+		for (final ItemMapping item : values())
+		{
+			for (int itemId : item.untradableItems)
+			{
+				MAPPINGS.put(itemId, item.tradeableItem);
+			}
+		}
+	}
+
+	ItemMapping(int tradeableItem, int... untradableItems)
+	{
+		this.tradeableItem = tradeableItem;
+		this.untradableItems = untradableItems;
+	}
+
+	/**
+	 * Get collection of items that are mapped from single item id.
+	 *
+	 * @param itemId the item id
+	 * @return the collection
+	 */
+	public static Collection<Integer> map(int itemId)
+	{
+		final Collection<Integer> mapping = MAPPINGS.get(itemId);
+
+		if (mapping == null || mapping.isEmpty())
+		{
+			return Collections.singleton(itemId);
+		}
+
+		return mapping;
+	}
+
+	/**
+	 * Map an item from its untradeable version to its tradeable version
+	 *
+	 * @param itemId
+	 * @return
+	 */
+	public static int mapOne(int itemId)
```

Comment:
This should be renamed to mapFirst so it properly indicates what the method actually does.